### PR TITLE
Add userstate to extra

### PR DESCRIPTION
**Pull request by informathemusic**
Add userstate to extra

### CODE REVIEW

File: dist/comfy.js [hunk 1]
```diff
@@ -361,6 +361,7 @@ var comfyJS = {
         var userColor = userstate[ "color" ];
         var emotes = userstate[ "emotes" ];
         var messageFlags = userstate[ "flags" ];
+	var messageUserstate
```
Comment (instafluff) [inline dist/comfy.js:364]: It looks like this line of code might not be needed

File: dist/comfy.js [hunk 2]
```diff
@@ -383,6 +384,7 @@ var comfyJS = {
           isEmoteOnly: isEmoteOnly,
           userId: userId,
           username: userstate[ "username" ],
+          userstate: userstate,
```
Comment (instafluff) [inline dist/comfy.js:387]: awesome!

**Review by instafluff — CHANGES_REQUESTED** [review]

File: dist/comfy.js (hunk 2 above)
Comment (informathemusic) [reply dist/comfy.js:387]: Oh ya i forgotz
:P

**Comment by informathemusic** [comment]
Here, changes made!

**Pull request merged**